azure-keyvault/tests/test_secrets_async.py

```diff
@@ -42,7 +42,7 @@ def run(test_class_instance, *args, **kwargs):
 class KeyVaultSecretTest(KeyvaultTestCase):
 
     def _assert_secret_attributes_equal(self, s1, s2):
-        self.assertEqual(s1.id , s2.id)
+        # self.assertEqual(s1.id , s2.id)
```

Comment:
thinking of removing this so that we can check attributes even when comparing a list object against a get object and since they only differ in `id`.